infrastructure/terraform/modules/deployment/main.tf

```diff
@@ -217,6 +217,7 @@ module "service_flock" {
        { name = "PGHOST", value = module.core_dependency_services.rds_connection_components.host },
        { name = "PGPORT", value = module.core_dependency_services.rds_connection_components.port },
        { name = "SUPABASE_URL", value = var.NEXT_PUBLIC_SUPABASE_URL },
+       { name = "PAGER", value = "less -S" },
```

Comment:
what is this?

Reply:
this is a convenience setting on the bastion host that improves (IMO) how psql handles long lines. the default pager is already `less` so this is just passing `-S` to it, which turns off wrapping. rn the wrapping is just unreadable for me:
<img width="1494" alt="image" src="https://github.com/user-attachments/assets/9fff0b2a-ed4e-424d-addf-fd186c3f031f">

but with `PAGER="less -S":
<img width="1487" alt="image" src="https://github.com/user-attachments/assets/0a89263b-8ff7-4c4a-a244-94aa8133dc6e">